### ISSUE TITLE
v0.9.0 no longer emits dev dependency targets

### ISSUE THREAD

**Issue by wchargin** [google/cargo-raze#355]
In `cargo-raze` v0.8.0 and prior, the user-facing `BUILD` package
included aliases for dev dependencies, just like normal dependencies. As
of v0.9.0, these seem to be removed entirely. This breaks tests and dev
tools.

Repro repo:
<https://github.com/wchargin/tensorboard/tree/wchargin-raze-v0.8.0>
in the `tensorboard/data/server/` directory:

Note first that at v0.8.0, the project builds fine:

```
$ cd tensorboard/data/server/
$ ~/tmp/cargo-raze-v0.8.0
$ git status
HEAD detached at wc/wchargin-raze-v0.8.0
nothing to commit, working tree clean
$ bazel test ...
INFO: Analyzed 13 targets (4 packages loaded, 50 targets configured).
INFO: Found 10 targets and 3 test targets...
INFO: Elapsed time: 1.506s, Critical Path: 0.04s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
//tensorboard/data/server:rustboard_core_doc_test               (cached) PASSED in 3.5s
//tensorboard/data/server:rustboard_core_test                   (cached) PASSED in 0.5s
//tensorboard/data/server:update_protos_test                    (cached) PASSED in 0.1s

Executed 0 out of 3 tests: 3 tests pass.
INFO: Build completed successfully, 1 total action
```

But when re-generating at v0.9.0, the configs are broken:

```
$ ~/tmp/cargo-raze-v0.9.0
$ git diff -U0
diff --git a/tensorboard/data/server/cargo/BUILD.bazel b/tensorboard/data/server/cargo/BUILD.bazel
index 1877cee4c..968f90e0f 100644
--- a/tensorboard/data/server/cargo/BUILD.bazel
+++ b/tensorboard/data/server/cargo/BUILD.bazel
@@ -114,9 +113,0 @@ alias(
@@ -168,9 +158,0 @@ alias(
@@ -214,2 +196,2 @@ alias(
@@ -222,6 +204,4 @@ alias(
@@ -228,0 +209 @@ alias(
-alias(
-    name = "prost_build",
-    actual = "@raze__prost_build__0_7_0//:prost_build",
-    tags = [
-        "cargo-raze",
-        "manual",
-    ],
-)
-
-alias(
-    name = "tempfile",
-    actual = "@raze__tempfile__3_1_0//:tempfile",
-    tags = [
-        "cargo-raze",
-        "manual",
-    ],
-)
-
-    name = "tonic_build",
-    actual = "@raze__tonic_build__0_4_0//:tonic_build",
-alias(
-    name = "walkdir",
-    actual = "@raze__walkdir__2_3_1//:walkdir",
-    tags = [
-        "cargo-raze",
-        "manual",
+    name = "walkdir",
+    actual = "@raze__walkdir__2_3_1//:walkdir",
+# Export file for Stardoc support
+exports_files(
+    [
+        "crates.bzl",
+    visibility = ["//visibility:public"],
diff --git a/third_party/rust/BUILD.bazel b/third_party/rust/BUILD.bazel
index 2b78032ca..0cad4cf6a 100644
Binary files a/third_party/rust/BUILD.bazel and b/third_party/rust/BUILD.bazel differ
diff --git a/third_party/rust/remote/BUILD.prost-build-0.7.0.bazel b/third_party/rust/remote/BUILD.prost-build-0.7.0.bazel
index 08e1efb1d..294e83a71 100644
Binary files a/third_party/rust/remote/BUILD.prost-build-0.7.0.bazel and b/third_party/rust/remote/BUILD.prost-build-0.7.0.bazel differ
$ bazel test ...
ERROR: /HOMEDIR/git/tensorboard/tensorboard/data/server/BUILD:69:10: no such target '//tensorboard/data/server/cargo:tempfile': target 'tempfile' not declared in package 'tensorboard/data/server/cargo' defined by /HOMEDIR/git/tensorboard/tensorboard/data/server/cargo/BUILD.bazel and referenced by '//tensorboard/data/server:rustboard_core_test'
ERROR: Analysis of target '//tensorboard/data/server:rustboard_core_test' failed; build aborted: Analysis failed
INFO: Elapsed time: 0.651s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (4 packages loaded, 11 targets configured)
FAILED: Build did NOT complete successfully (4 packages loaded, 11 targets configured)
```

I checked the v0.8.0...v0.9.0 shortlog but didn’t see anything to
suggest that this was intentional (and there are no release notes or
changelogs, so I can’t infer from that). None of the `examples/` have
dev dependencies, so this wouldn’t be caught there. 

Presumably this is unintentional—else, how am I meant to run tests?

(I can try to bisect later today if helpful.)

**Comment by wchargin** [comment]
First bad commit: 9d01e8933c9254a791bc973a20d9aa62ba266b17 (#319).

**Comment by UebelAndre** [comment]
Oof, sorry about that. I'll take a look later today. Definitely did not intend to break dev dependencies.

**Comment by UebelAndre** [comment]
I think https://github.com/google/cargo-raze/pull/358 should unblock you. I'm going to do more testing and add some tests to go with the new examples once I feel confident about this change.

**Comment by wchargin** [comment]
Applying that patch seems to generate the right build files. I’m not
actually unblocked until it’s merged and released; we use `cargo-raze`
on our team and CI, and it’s not reasonable for me to ask everyone to
install a patched version from your fork. But I’ll stand by; thanks!

**Comment by UebelAndre** [comment]
Totally understand. I'll do everything I can to help get CI working and this change merged and deployed 😄 

Again, sorry for breaking your workflow! 😅

**Comment by dfreese** [comment]
fixed in 0.9.2

**Comment by wchargin** [comment]
Lovely; thanks to everyone involved for the quick release!
